sample/build.gradle.kts

```diff
@@ -69,6 +70,6 @@ dependencies {
     implementation(Dependencies.Compose.activity)
     implementation(Dependencies.Compose.navigation)
     implementation(Dependencies.Compose.tooling)
-
+    implementation("io.coil-kt:coil-compose:2.0.0-rc01")
```

Comment:
Shouldn't be necessary (and I think it isn't).

Reply:
Yes, I was using it in my sample, but I found another way